content/3_Silver/Prefix_Sums.problems.json

```diff
@@ -160,6 +160,19 @@
       "solutionMetadata": {
         "kind": "internal"
       }
+    },
+    {
+      "uniqueId": "usaco-600",
+      "name": "USACO January 2016 Platinum - Fort Moo",
```

Comment:
why is the name part different from other problems?

Reply:
For problem suggestions, you should just update it to be `Fort Moo` and merge it.